src/examples/bsl_wp_smt/mem-gauss-test.sml

```diff
@@ -0,0 +1,174 @@
+open HolKernel Parse boolLib bossLib;
+open bslSyntax;
+open pretty_exnLib;
+
+(* Load the dependencies in interactive sessions *)
```

Comment:
`easy_noproof_wpLib` isn't needed, because it should be in the heap ("should" because if you look into the `Hoolmakefile.gen`, you'll see that it's not (yet)). But I need to load `HolSmtLib` because vim doesn't auto-load files. I think that I stole this pattern (load in `if !Globals.interactive` in `HolSmtLib` :innocent: )